Source/Scene/Scene.js

```diff
@@ -2562,6 +2562,9 @@ define([
 
         us.update(frameState);
 
+        var backgroundColor = defaultValue(scene.backgroundColor, Color.BLACK);
+        us.updateBackgroundColor(backgroundColor);
```

Comment:
Store `backgroundColor` on `FrameState` and have `UniformState.backgroundColor.get` return `this.frameState.backgroundColor`, or store a second copy on `UniformState`?